sympy/matrices/matrices.py

```diff
@@ -4168,10 +4168,10 @@ def QRdecomposition(self):
     def QRsolve(self, b):
         """Solve the linear system 'Ax = b'.
```

Comment:
This could be improved as well.

Reply:
I've replaced every instance of 'Ax = b' with ``Ax = b``, please review!!

Reply:
`Ax = b` isn't valid Python code. It should be left as-is, or changed to a math environment (:math:\`Ax = b\`).